From 8cbcf518cc478ca0b8199704f523c1495da400c4 Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Thu, 13 Sep 2012 17:33:41 +0800
Subject: ENGR00219882 wm8962: avoid the digital volume changed by user

Wolfson suggests that digital volume should not be changed by user
to avoid affecting SNR and dynamic range, and user should adjust
analog volume.

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/codecs/wm8962.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/codecs/wm8962.c b/sound/soc/codecs/wm8962.c
index b145311..784b595 100644
--- a/sound/soc/codecs/wm8962.c
+++ b/sound/soc/codecs/wm8962.c
@@ -2218,8 +2218,6 @@ SOC_ENUM("Capture LHPF Mode", cap_lhpf_mode),
 SOC_DOUBLE_R_TLV("Sidetone Volume", WM8962_DAC_DSP_MIXING_1,
 		 WM8962_DAC_DSP_MIXING_2, 4, 12, 0, st_tlv),
 
-SOC_DOUBLE_R_TLV("Digital Playback Volume", WM8962_LEFT_DAC_VOLUME,
-		 WM8962_RIGHT_DAC_VOLUME, 1, 127, 0, digital_tlv),
 SOC_SINGLE("DAC High Performance Switch", WM8962_ADC_DAC_CONTROL_2, 0, 1, 0),
 
 SOC_SINGLE("ADC High Performance Switch", WM8962_ADDITIONAL_CONTROL_1,
-- 
1.7.9.5

